Add event integration template to ddev (#21717)
DataDog/integrations-core: datadog_checks_dev/datadog_checks/dev/tooling/templates/integration/event/README.md

# Event Integration

Choose this type of integration if your integration primarily sends events to Datadog.

This template is for integrations that focus on event collection.

> There is no Python package for this type of integration, it essentially contains a manifest.

Reviewer: ```suggestion
There is no Python package for this type of integration, only a manifest is essential.
```

Reviewer: Not sure I'm grasping the meaning of `it essentially contains a manifest`, so please let me know if this is inaccurate.
